components/org.wso2.carbon.identity.application.authenticator.fido2/src/main/java/org/wso2/carbon/identity/application/authenticator/fido2/dao/FIDO2DeviceStoreDAO.java

```diff
@@ -549,6 +561,9 @@ public void addFIDO2RegistrationByUsername(String username, FIDO2CredentialRegis
                         .PUBLIC_KEY_COSE));
                 Long signatureCount = resultSet.getLong(FIDO2AuthenticatorConstants.SIGNATURE_COUNT);
                 Timestamp timestamp = resultSet.getTimestamp(FIDO2AuthenticatorConstants.TIME_REGISTERED);
+                boolean isUsernamelessSupported =
+                        "1".equals(resultSet.getString(FIDO2AuthenticatorConstants.IS_USERNAMELESS_SUPPORTED));
```

Comment:
Shall we define a constant for 1, to make it more understandable? `USERNAMELESS_SUPPORTED` maybe?